packages/frontend/src/components/DashboardTiles/TileForms/AddChartTilesModal.tsx

```diff
@@ -100,6 +100,10 @@ const AddChartTilesModal: FC<Props> = ({ onAddTiles, onClose }) => {
                         chartName: chart?.name ?? '',
                     },
                     type: DashboardTileTypes.SAVED_CHART,
+                    tabUuid: (
+                        dashboard?.tabs.find((t) => t.isDefault) ??
+                        dashboard?.tabs?.[0]
+                    )?.uuid,
```

Comment:
Not worth having this logic as we set the tabUuid in `handleAddTiles` (Dashboard.tsx)

```suggestion
                    tabUuid: undefined,
```

Reply:
will do 👍